step1_SMS-PM/profile_columns.list

```diff
@@ -0,0 +1,962 @@
+! profile_columns.list -- determines the contents of star model profiles
```

Comment:
Again, do we need this file? Why can't we use the profile_columns_common.list for this step?